### PR TITLE
Give upload permission

### PR DESCRIPTION
**Pull request by pramodanarase**
# Description
Please give upload permission for CA APM Plugin hosted on https://github.com/jenkinsci/ca-apm-plugin. Along with @Srikant Srikant

I am also a committer for this repository. My community account name is @pramodanarase

### Always

- [x] Add link to plugin/component Git repository in description above - done

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions) - done
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name - 
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by batmat** [comment]
Please check the right boxes in the description. Thanks.

**Comment by pramodanarase** [comment]
@batmat - done.

**Comment by batmat** [comment]
Given I see the submitter already in commits in the repo, I think this looks good. But as permission to release is slightly different that commit one, we're now waiting on the existing releaser to approve this. Thanks!

**Comment by batmat** [comment]
(Also, we'll wait for a PR build, but the CI is currently not a great shape...)

**Comment by srikns** [comment]
@batmat - I am the original releaser and I approve @pramodanarase to be additional releaser

Thanks

**Comment by batmat** [comment]
Checked locally, no error, merging.

**Comment by batmat** [comment]
@daniel-beck can you please trigger a manual build of all current PRs so we know the status? Thanks

**Comment by pramodanarase** [comment]
@daniel-beck - Can you please look into this we blocked on this?

File: permissions/plugin-ca-apm.yml [hunk 1]
```diff
@@ -3,5 +3,7 @@ name: "ca-apm"
 github: "jenkinsci/ca-apm-plugin"
 paths:
 - "org/jenkins-ci/plugins/ca-apm"
+#added new developer
```
Comment (daniel-beck) [inline permissions/plugin-ca-apm.yml]: Huh?

This is the job of the SCM's commit history.

Comment (batmat) [reply permissions/plugin-ca-apm.yml]: It was added just an attempt from the submitter to nudge the PR build, FWIW. I should have removed it myself before merging.

**Pull request merged**